### ISSUE TITLE
Thenar muscle aplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11686]
# Thenar muscle aplasia

## Proposed definition

Complete congenital absence of the thenar muscle, distinguished from thenar muscle hypoplasia (which is reduced size).

## Parent term

- `HP:0001227 Abnormality of the thenar eminence`

## Synonyms

- Thenar eminence aplasia

## Associated diseases

_None provided_

## Contributor

- ORCID: 0009-0001-4998-8617

**Comment by pnrobinson** [comment]
@evanc1648 do you have a PMID. Clinically, the apparent absence of a thenar muscle cannot reliably be distinguished from sever hypoplasia or aplasia, so we will need a definition that specifies how to diagnosis this clinically.

**Comment by evanc1648** [comment]
Thanks @pnrobinson! I found "Congenital hypoplastic thumb with absent thenar muscles" ([PMID:6491208](https://pubmed.ncbi.nlm.nih.gov/6491208/)): a case report documenting absent thenar muscles alongside a congenital hypoplastic thumb. Our finding was recorded as "absence of the thenar muscle," and the FA guidelines describe Type II thumb deficiency as involving "absence of the thenar muscle at the base of the thumb." The modified Blauth classification lists the same--Type II as absence of the intrinsic thenar muscles ([PMID:33402764](https://pubmed.ncbi.nlm.nih.gov/33402764/), this article shows the classification table and cites [Blauth's original article](https://link.springer.com/article/10.1007/BF00416751)).

Since absence and severe hypoplasia can be hard to tell apart during an exam, would it make sense to reframe this as a combined "Aplasia/Hypoplasia of the thenar eminence" term, similar to how HPO handles the thumb with Aplasia/Hypoplasia of the thumb (HP:0009601)? Otherwise, we're happy to withdraw and map to the existing Small thenar eminence (HP:0001245) on our end. Thank you!

**Comment by pnrobinson** [comment]
@evanc1648 reviewing the PMIDs, it think it is justified to make a term. Thanks!

Adding new term: 
- ID: HP:6001527
- label: Thenar muscle aplasia
- parents: [Abnormality of the thenar eminence (HP:0001227)](https://hpo.jax.org/app/browse/term/HP:0001227)
- pmid: PMID:33402764
- def: Complete congenital absence of the thenar muscle.
- comment: 
- orcid: orcid:0009-0001-4998-8617
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11686
- synonyms:
